modules/Quiz/resources/quiz.js

```diff
@@ -937,12 +971,18 @@
                     .text(gM('mwe-quiz-selected'))
                     .addClass('qApplied').fadeIn(100).attr('aria-disabled', true);
             });
-            _this.KIVQModule.submitAnswer(questionNr,_this.selectedAnswer);
-            _this.selectedAnswer = null;
-            $(".cp-navigation").css("pointer-events", "none");
-            $(".ftr-right").css("pointer-events", "none");
-            $(".cp-navigation-btn").addClass("disabled");
-            setTimeout(function(){_this.KIVQModule.checkIfDone(questionNr)},_this.postAnswerTimer);
+            _this.KIVQModule.submitAnswer(questionNr,_this.selectedAnswer).then(
```

Comment:
Are we doing this on all questions types? Should we not do this only on reflection-point questions?

Reply:
logic changed according to new requirements about reflection point